diff --git a/Source/Core/Common/MemArenaUnix.cpp b/Source/Core/Common/MemArenaUnix.cpp
index 1743c2ace0..311469f1f0 100644
--- a/Source/Core/Common/MemArenaUnix.cpp
+++ b/Source/Core/Common/MemArenaUnix.cpp
@@ -27,7 +27,7 @@ MemArena::~MemArena() = default;
 
 void MemArena::GrabSHMSegment(size_t size)
 {
-  const std::string file_name = "/dolphin-emu." + std::to_string(getpid());
+  const std::string file_name = "/snap.dolphin-emulator.dolphin-emu." + std::to_string(getpid());
   fd = shm_open(file_name.c_str(), O_RDWR | O_CREAT | O_EXCL, 0600);
   if (fd == -1)
   {
